game/scripts/npc/abilities/clinkz_burning_army.txt

```diff
@@ -50,7 +50,7 @@
 			"04"
 			{
 				"var_type"				"FIELD_FLOAT"
-				"attack_rate"			"1.75 1.6 1.45 1.25 1.0"
+				"attack_rate"			"1.75 1.6 1.45 1.25 1.1"
```

Comment:
I suggest "1.75 1.6 1.45 1.3 1.15"

Reply:
Its funny how they started to pick Clinkz after the minor. And everyone told me Burning Army is terrible (because its base dmg only).